hiring/job-adverts/hiring-process.pug

```diff
@@ -0,0 +1,109 @@
+
+    h2 The hiring process
+
+    p It takes approximately four to six weeks from the application closing
+        |  date to us making a job offer – but we’ll try to make it faster if we can.
+
+    p We focus on skills, experience, behaviours and potential – we won’t ask any trick
+        |  questions, and we don’t expect you to know anything to a textbook level of detail.
+
+    .list
+        ol
+            li CV and covering letter
+                p The first stage of the application process is a CV and covering letter.
+                    |  At this stage we want to find out more about you, your background, what 
+                    |  interests and motivates you, and why you feel you’ll be a good fit at ONS.
+                p As a rough guide, your CV should be around two to three A4 pages, and your
+                    |  covering letter around one A4 page – but there’s no strict limits.
+                p We’ll let you know if you’ve been successful within a few days of the closing
+                    |  date.
+
+            li Telephone interview
+                p If you’re successful at the application stage we’ll invite you to a 30-minute
+                    |  telephone interview. This will be a technical interview to explore your
+                    |  skills and capability in a bit more detail.
+                p We’ll ask you for an example of some difficult technical work you’ve done
+                    |  recently. This could be a challenging coding problem or a complex system
+                    |  requirement. We’ll ask some follow up questions to understand your technical
+                    |  capability.
+                p We’ll typically conduct our telephone interviews over a week, so you might
+                    |  have to wait a few days to hear if you’ve been successful.
+
+            li Invitation to interview
+                p If you’re successful at the telephone interview stage we’ll invite you to an
+                    |  on-site interview at ONS in Newport.
+                p You shouldn’t need to prepare for this – we’re not testing your ability to 
+                    |  remember things, but it’s worth reviewing your CV and covering letter, and
+                    |  thinking about some examples of the work you’ve been doing recently.
+
+            li Your interview
+                p Your interview will be on-site at ONS in Newport. It’ll take around two to three
+                    |  hours, and will be a technical and behavioural interview, followed by
+                    |  meeting some other members of the team. The interview is based on the 
+                    |  <a href="https://www.gov.uk/government/publications/success-profiles">Civil Service Success Profiles</a>.
+                p We know three hours is a long time, so we’ll make sure we give you plenty of
+                    |  opportunities to take a break. We’ll also make sure there’s pens, paper and
```

Comment:
Believe this should be `make sure there're pens`-  there are as its multiple not there is